plugin-dev/Source/Sentry/Private/SentrySubsystem.cpp

```diff
@@ -12,15 +12,13 @@
 #include "Misc/EngineVersion.h"
 #include "Misc/CoreDelegates.h"
 
+#include "Interface/SentrySubsystemInterface.h"
```

Comment:
Adding this should be enough to resolve the missing forward declaration issue. The same approach is used for some entities that are supported only on mobile platforms and that doesn't break the compilation on desktop.